src/main/java/az/gdg/mscomplaint/model/entity/ComplaintStatusEntity.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package az.gdg.mscomplaint.model.entity;
```

Comment:
remove it. If I write a comment about smth one time, you should apply it everywhere in that pr.